CI issues due to fetch_openml of iris

There still are occasional CI errors due to failing to fetch Iris dataset from OpenML (https://github.com/ramp-data/iris/issues/1). It's transient so restaring the CI later usually fixes it.

Related to https://github.com/openml/OpenML/issues/1092

The error message is also very confusing as it would show,
```
                raise CalledProcessError(retcode, process.args,
>                                        output=stdout, stderr=stderr)
E               subprocess.CalledProcessError: Command '['python', 'prepare_data.py']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

database_connection = None

    @pytest.fixture
    def session_toy(database_connection):
        database_config = read_config(database_config_template())
        ramp_config = ramp_config_template()
        try:
            deployment_dir = create_toy_db(database_config, ramp_config)
            with session_scope(database_config['sqlalchemy']) as session:
                yield session
        finally:
>           shutil.rmtree(deployment_dir, ignore_errors=True)
E           UnboundLocalError: local variable 'deployment_dir' referenced before assignment
/home/runner/work/ramp-board/ramp-board/ramp-engine/ramp_engine/tests/test_dispatcher.py:39: UnboundLocalError
---------------------------- Captured stderr setup -----------------------------
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/testenv/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 414, in _get_data_info_by_name
    url, error_message=None, data_home=data_home
  File "/usr/share/miniconda/envs/testenv/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 179, in _get_json_content_from_openml_api
    raise OpenMLError(error_message)
sklearn.datasets._openml.OpenMLError: None
 During handling of the above exception, another exception occurred:
[...]
    raise OpenMLError(error_message)
sklearn.datasets._openml.OpenMLError: Dataset iris with version 1 not found.
```

So there is a second bug in pytest fixtures about `deployment_dir` badly handled when `create_toy_db` fails but that's not too critical.

--- a/prepare_data.py
+++ b/prepare_data.py
@@ -1,7 +1,7 @@
 import os
 
 import click
-from sklearn.datasets import fetch_openml
+from sklearn.datasets import load_iris
 from sklearn.model_selection import train_test_split
 
 PATH_DATA = os.path.join(
@@ -11,21 +11,19 @@ PATH_DATA = os.path.join(
 
 @click.command()
 def prepare_data():
-    data = fetch_openml(
-        "iris", version=1, as_frame=True, data_home=PATH_DATA
-    )
+    data = load_iris(as_frame=True)
     df = data.frame
     df = df.rename(columns={
-        "sepallength": "sepal length",
-        "sepalwidth": "sepal width",
-        "petallength": "petal length",
-        "petalwidth": "petal width",
-        "class": "species",
+        "sepal length (cm)": "sepal length",
+        "sepal width (cm)": "sepal width",
+        "petal length (cm)": "petal length",
+        "petal width (cm)": "petal width",
+        "target": "species",
     })
     df["species"] = df["species"].replace({
-        "Iris-setosa": "setosa",
-        "Iris-versicolor": "versicolor",
-        "Iris-virginica": "virginica"
+        0: "setosa",
+        1: "versicolor",
+        2: "virginica"
     })
 
     # 1- split into private train and test sets.

